tests/integration/models/reddit/test_subreddit.py

```diff
@@ -1599,7 +1599,7 @@ def test_update(self, _):
         with self.recorder.use_cassette("TestSubredditModeration.test_update"):
             before_settings = self.subreddit.mod.settings()
             new_title = before_settings["title"] + "x"
-            if len(new_title) == 20:
+            if len(new_title) >= 20:
                 new_title = "x"
             self.subreddit.mod.update(title=new_title)
             assert self.subreddit.title == new_title
```

Comment:
It would seem that this test somehow got out of sync so the assertion is broken. Otherwise, this PR looks great.

Reply:
It's the placeholder value. With environment variables, it works perfectly.